forex-service/src/main/java/lk/ac/ucsc/forexservice/ExchangeValue.java

```diff
@@ -0,0 +1,77 @@
+package lk.ac.ucsc.forexservice;
+
+import javax.persistence.Id;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "Exchange_Value")
+public class ExchangeValue {
+
+    @Id
+    private Long id;
```

Comment:
@Binurim why long? but not double?

Reply:
No specific reason. Are we generally use double?